### PR TITLE
doc: added section about retival of Service Specication

### PR DESCRIPTION
**Pull request by wornet-mwo**
Signed-off-by: Michael Wodniok <wodniok@wor.net>

Just updated doc a little bit, as i had to ask in the mailing list.

### CODE REVIEW

**Comment by sebastian-philipp** [comment]
Hm. `--export` is already mentioned in https://github.com/ceph/ceph/blob/84bdafe0e36e11fb8b57065625a5ecb49fd3c248/doc/mgr/orchestrator.rst#service-status

do you want to extend this section instead?

**Comment by wornet-mwo** [comment]
I understand your point.

We could move this example but i would add a link in the other section as the page is quite long and the topic simply matches more the latter "Service Specification" (how to build a spec and import it + the new section how to export it again).

**Comment by sebastian-philipp** [comment]
> I understand your point.
> 
> We could move this example but i would add a link in the other section as the page is quite long and the topic simply matches more the latter "Service Specification" (how to build a spec and import it + the new section how to export it again).

ok!

**Comment by wornet-mwo** [comment]
Just to be clear should i

A: move and link from Service Spec to Status
or
B: link from Status to Service Spec
?

**Comment by sebastian-philipp** [comment]
B sounds good to me!

File: doc/mgr/orchestrator.rst [hunk 1]
```diff
@@ -575,6 +575,18 @@ Many service specifications can be applied at once using
       all: true
     EOF
 
+Retrieving the running Service Specification
+--------------------------------------------
+
+If started via ``ceph orch apply <rgw|mds|...>`` changing the Service
+Specification directly is normally a bit more complicated. It is easier to
+export the running Service Specification like so::
```
Comment (sebastian-philipp) [inline doc/mgr/orchestrator.rst:583]: @zdover23 can we rephrase this to sound a bit less German?

File: doc/mgr/orchestrator.rst [hunk 2]
```diff
@@ -575,6 +575,18 @@ Many service specifications can be applied at once using
...
+    
+    ceph orch ls rgw rgw.<id> --export yaml >rgw.<id>.yaml
+    ceph orch ls mgr mgr --export yaml >mgr.yaml
```
Comment (sebastian-philipp) [inline doc/mgr/orchestrator.rst:586]:
```suggestion
    ceph orch ls --service-name rgw.<daemon-id> --export > rgw.<daemon-id>.yaml
    ceph orch ls --service-type mgr --export > mgr.yaml
    ceph orch ls --export > cluster.yaml
```

**Review by sebastian-philipp — CHANGES_REQUESTED** [review]

File: doc/mgr/orchestrator.rst (hunk 1 above)
Comment (wornet-mwo) [reply doc/mgr/orchestrator.rst:583]: School-english...

```
If services have been started using ``ceph orch apply ...`` without specification file changing it is still possible. An easy way to do so is exporting the running Service Specification::
```

Comment (zdover23) [reply doc/mgr/orchestrator.rst:586]: I suggest this:

BEGIN QUOTED TEXT
If the services have been started via ``ceph orch apply...``, then directly changing the Services Specification is complicated. Instead of attempting to directly change the Services Specification, we suggest exporting the running Service Specification by following these instructions:
END QUOTED TEXT

Comment (wornet-mwo) [reply doc/mgr/orchestrator.rst]: Ok, thank you @zdover23. The change has been applied.

**Review by sebastian-philipp — APPROVED** [review]
